### DISCUSSION TITLE
Slack connect channel for Surface OSS

### DISCUSSION THREAD

**Discussion by bogdanoniga** [surface-security/surface#116] [General]
Would be nice to have a Slack connect channel for Surface OSS where to address extensive discussions or even for jumping in huddles if needed. What's your view on this?

**Comment by gsilvapt** [comment]
Yes, we've been discussing this idea but there are honestly too many pros and cons in the multiple platforms to just choose one thing.
- I'm okay with Slack, it's easy to maintain and use, I guess. If we want to open it to the world, it would still be easy to manage permissions/visibility of things. The free plan should suffice for our use case.
- Discord is fine-ish, although more complex to maintain but that matters in terms of publicity - and I think the project's focus is likely different at this stage.
- Telegram/Signal/WhatsApp or something similar could work but I don't think it scales.

I'm okay with using GitHub Discussions, although this means we have to be extra careful with what we share with each other :) For any meeting need, we can always schedule something or invest heavily in asynchronous communication.
